react/ProductList.tsx

```diff
@@ -24,7 +24,7 @@ export function getProductList(products?: Product[]) {
     '@type': 'ListItem',
     position: index + 1,
     name: product.productName,
-    url: `${baseUrl}/${product.linkText}`,
+    url: `${baseUrl}/${product.link}`,
```

Comment:
Just to make this safer, since this function is exported and could be used by any IO app:

Create a new variable inside this function call and generate the URL like this:

```
const productUrl = `${baseUrl}/${product.link ?? product.linkText + '/p'}`
```
What do you think?

Reply:
@victorhmp we have implemented the changes you requested